web/components/templates/keys/addKeyModal.tsx

```diff
@@ -46,21 +47,35 @@ const AddKeyModal = (props: AddKeyModalProps) => {
       dashes: true,
     }).toString()}`.toLowerCase();
     setReturnedKey(apiKey);
-    hashAuth(apiKey).then((res) => {
-      supabaseClient
-        .from("helicone_api_keys")
-        .insert({
-          api_key_hash: res,
-          user_id: user?.id!,
-          api_key_name: keyName.value,
-          organization_id: org?.currentOrg?.id!,
-          created_at: new Date(),
-        })
-        .then(() => {
-          setNotification("Successfully created API key", "success");
-          setIsLoading(false);
-          onSuccess();
-        });
+
+    const authFromCookie = getHeliconeCookie();
+    const resp = await fetch(
```

Comment:
Make sure to deploy jawn before merging otherwise anyone that generates a key will fail